### PR TITLE
add Admin section of frontend

### PR DESCRIPTION
**Pull request by lucaspreel**

### CODE REVIEW

File: microservices/frontend/src/modules/admin/AdminPage.js [hunk 1]
```diff
@@ -1,4 +1,59 @@
...
+    } catch (e) {
+      setState((state) => ({ ...state, loaded: true }));
+      toast.error(`Error al borrar el usario. ${e.message}`);
+    }
+  }
+
+  async function onUserEdit (newUser, user) {
+    console.log(newUser);
+    console.log(user);
+    //setState((state) => ({ ...state, loaded: false }));
```
Comment (0aps) [inline microservices/frontend/src/modules/admin/AdminPage.js]: remove death code. i.e comments you're not using.

Comment (0aps) [reply microservices/frontend/src/modules/admin/AdminPage.js]: also, remove consoles logs

**Comment by 0aps** [comment]
The rest of the changes looks good to me. Good job.

File: microservices/frontend/src/modules/admin/AdminPage.js (hunk 1 above)
Comment (lucaspreel) [reply microservices/frontend/src/modules/admin/AdminPage.js]: Done :)

**Pull request merged**